guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1095 @@
+
+Some commonly used naming convention in Discord.js have been changed:
+
+#### Thing#thingId
+
+The casing of `thingID` properties has changed to `thingId`. This is a more-correct casing for the camelCase used by Discord.js as `Id` is an abbreviation, not an acronym.
+
+```diff
+- console.log(guild.ownerID);
++ console.log(guild.ownerId);
+
+- console.log(interaction.channelID);
++ console.log(interaction.channelId);
+```
+
+#### Client#message
+
+#### Client#interaction
+
+- `message` and `interaction` events have been renamed to `messageCreate` and `interactionCreate` respectively, to bring the library in line with Discord's naming conventions.
+
+Don't worry - the old names still work, but you'll receive a Deprecation Warning until you switch over.
+
+```diff
+- client.on("message", message => { ... });
++ client.on("messageCreate", message => { ... });
+
+- client.on("interaction", interaction => { ... });
++ client.on("interactionCreate", interaction => { ... });
+```
+
+### Snowflakes
+
+For TypeScript users, discord.js now enforces the `Snowflake` type, a stringified bigint, rather than allowing any string to be accepted.
+
+```diff
+interface Config {
+	prefix: string;
+- ownerId: string;
++ ownerId: Snowflake;
+}
+```
+
+### Allowed Mentions
+
+`clientOptions.disableMentions` has been removed and replaced with `clientOptions.allowedMentions`!
+
+The Discord API now allows bots much more granular control over mention parsing, down to the specific id.
+
+Refer to the [Discord API documentation](https://discord.com/developers/docs/resources/channel#allowed-mentions-object) for more information.
+
+```diff
+- const client = new Discord.Client({ disableMentions: 'everyone' });
++ const client = new Discord.Client({ allowedMentions: { parse: ['users', 'roles'], repliedUser: true } });
+```
+
+### Replies / Message#reply
+
+`Message.reply()` will no longer result in the bot prepending a user mention to the content, replacing the behavior with Discord's reply feature.
+
+`MessageOptions.reply` no longer takes a user id. It has been replaced with a `ReplyOptions` type, expecting `MessageOptions.reply.messageReference` as a Message id.
+
+```diff
+- channel.send('content', { reply: '123456789012345678' }); // User id
++ channel.send({ content: 'content', reply: { messageReference: '765432109876543219' }}); // Message id
+```
+
+The new `MessageOptions.allowedMentions.repliedUser` boolean option determines if the reply will notify the author of the original message.
+
+```diff
+- message.reply('content')
++ message.reply({ content: 'content', allowedMentions: { repliedUser: false }})
+```
+Note that this will disable all other mentions in this message. To enable other mentions, you will need to include other `allowedMentions` fields. See the above "Allowed Mentions" section for more.
+
+### Bitfields / Permissions
+
+Bitfields are now `BigInt`s instead of `Number`s. This can be handled using the `BigInt()` class, or the n-suffixed BigInt literal.
```

Comment:
```suggestion
Bitfields are now `BigInt`s instead of `Number`s. This can be handled using the `BigInt()` class, or the n-suffixed [BigInt literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt).
```